platforms-serverless/firebase-functions/test.sh

```diff
@@ -7,9 +7,9 @@ url="https://us-central1-prisma-e2e-tests-265911.cloudfunctions.net/$func"
 prisma_version="$(cat ../../.github/prisma-version.txt)"
 
 if [ "$PRISMA_CLIENT_ENGINE_TYPE" == "binary" ]; then
-  files=',"files":["deno","edge.d.ts","edge.js","index-browser.js","index.d.ts","index.js","package.json","query-engine-debian-openssl-1.1.x","schema.prisma"]'
+  files=',"files":["deno","edge.d.ts","edge.js","index-browser.js","index.d.ts","index.js","package.json","query-engine-debian-openssl-1.0.x","query-engine-debian-openssl-1.1.x","schema.prisma"]'
 else
-  files=',"files":["deno","edge.d.ts","edge.js","index-browser.js","index.d.ts","index.js","libquery_engine-debian-openssl-1.1.x.so.node","package.json","schema.prisma"]'
+  files=',"files":["deno","edge.d.ts","edge.js","index-browser.js","index.d.ts","index.js","libquery_engine-debian-openssl-1.0.x.so.node","libquery_engine-debian-openssl-1.1.x","package.json","schema.prisma"]'
```

Comment:
Should this really include both now?

Reply:
This is because `binaryTargets = ["native", "debian-openssl-1.1.x"]` was defined in the original schemas (see [binary](https://github.com/prisma/ecosystem-tests/blob/982fb472e74c00d8ae42148262ef115ad61122a8/platforms-serverless/firebase-functions/functions/prisma/schema-with-binary.prisma#L3), [node-api](https://github.com/prisma/ecosystem-tests/blob/982fb472e74c00d8ae42148262ef115ad61122a8/platforms-serverless/firebase-functions/functions/prisma/schema-with-node-api.prisma#L3)). I bet if I remove that `binaryTargets` line, we'd only see one `libquery` engine in the `files` expectation in `test.sh`

Reply:
In logs both are present indeed, that's new and unexpected to me

Reply:
For firebase and Google Cloud functions (Using Ubuntu 18.04 image like in our setup) in general the expected OpenSSL version is `openssl 	1.1.1-1ubuntu2.1~18.04.19` so `debian-openssl-1.1.x` which means for the QR `query-engine-debian-openssl-1.1.x`
https://cloud.google.com/functions/docs/reference/system-packages#ubuntu_1804

This mentions that Ubuntu 18.04 is used for `nodejs14` runtime which we use in these tests
https://cloud.google.com/functions/docs/concepts/execution-environment

It's unexpected that `1.0.x` shows up here to me

Reply:
So the native detection changed from also detecting `debian-openssl-1.1.x` to now `debian-openssl-1.0.x`?

Reply:
@Jolg42 this link says both OpenSSL 1.0 (1.0.2n-1ubuntu5.10) and OpenSSL 1.1 (1.1.1-1ubuntu2.1~18.04.19) are installed.
<img width="577" alt="image" src="https://user-images.githubusercontent.com/4923335/212692214-5f83ef0c-2e1f-4e77-8c9a-a8296ef6fd2f.png">

`openssl` package is OpenSSL CLI, and it indeed is only present for OpenSSL 1.1, while OpenSSL 1.0 exists only as a library, but the library and not the CLI binary is what's important for us.

Reply:
@janpio we now prefer the oldest among the installed versions since https://github.com/prisma/prisma/pull/17241

Reply:
Ok, then this change makes sense. Why do we even have `binaryTargets` in this test? Shouldn't we be able to remove it and update the test expectations to just the newly selected one and it should work?

Reply:
I second what @aqrln has written. @janpio, the CI is green now

Reply:
Note: binaryTargets were removed https://github.com/prisma/ecosystem-tests/pull/3373#pullrequestreview-1250314429